pysat/utils/_core.py

```diff
@@ -43,8 +42,7 @@ def set_data_dir(path=None, store=None):
 
 def computational_form(data):
     """
-    Input Series of numbers, Series, or DataFrames repackaged
-    for calculation.
+    Repackages numbers, Series, or DataFrames
```

Comment:
How about:

Repackages numbers, Series, or DataFrames

Regardless of input format, mathematical operations may be performed on the output via the same pandas mechanisms.
::
   mean = computational_form(data).mean()

Reply:
The :: was me trying to get the code formatting in docstring

Reply:
Perhaps a note being more specific on how and why things are being repackaged, so I know when I'm supposed to be using this.  Because right now I don't know when this would be useful.

Reply:
Good point.

This method may be particularly useful in analysis methods that aim to be instrument independent. pysat.Instrument objects can package data in a variety of ways within a DataFrame, depending upon the scientific data source. Thus, a variety of data types will be encountered by instrument independent methods and computational_form method may reduce the effort required to support more generalized processing.

Reply:
added